lib/inspec/rspec_extensions.rb

```diff
@@ -65,14 +65,30 @@ def method_missing(method_name, *arguments, &block)
 
 class RSpec::Core::ExampleGroup
   # This DSL method allows us to access the values of inputs within InSpec tests
-  def attribute(name)
-    Inspec::InputRegistry.find_or_register_input(name, self.class.metadata[:profile_id]).value
+  def input(input_name, options = {})
```

Comment:
Maybe add `options[:value] ||= name` to DRY up your tests below and keep it API compatible to the old name?

Reply:
(where "below" means in the whole diff, not in this commit... that confused myself :P)

Reply:
Do you mean `options[:value] ||= value` ?

Reply:
Ah. Sorry for being unclear. If you default `options[:value] ||= input_name` (my original comment had `name`, confusingly), then that can clean up your diffs below and dry it up a bit:

```diff
-  subject { attribute('test-03') } 
+  subject { input('test-03') }
```

Reply:
It is valid for an input (which is a user-facing DSL feature) to have an specified value, as well as it to have an explicit nil or an explicit false. Defaulting to the name of the input would be very surprising behavior; I don't think it's appropriate.

I'm sure the tests could probably be DRY'd up.